common/lib/xmodule/xmodule/js/src/video/06_video_progress_slider.js

```diff
@@ -75,6 +75,16 @@ function() {
     //     via the 'state' object. Much easier to work this way - you don't
     //     have to do repeated jQuery element selects.
     function _renderElements(state) {
+        if (state.config.enableProgressBar === false) {
```

Comment:
Instead of emptying html template we should stop load the progress or speed controls we they are disabled.
See how we disable auto advance control [here](https://github.com/edx/edx-platform/blob/3152c14192dcccac79b7b7a76eba91b380eba78a/common/lib/xmodule/xmodule/js/src/video/10_main.js#L95)